### DISCUSSION TITLE
Deleting an lazy loaded relationship does not remove the foreign key from model

### DISCUSSION THREAD

**Discussion by ApetersDev** [CodeDredd/pinia-orm#1968] [Q&A]
Is it expected that the foreign key is not removed from model after destroying the lazy loaded related object? The relation is null but the foreignkey is only removed in store. Everything works fine, if the relation is loaded directly.

Example with a relation between alliance and player:
Sandbox: https://codesandbox.io/p/sandbox/dv56qn

**Comment by CodeDredd** [comment]
I tried to open your sandbox example, but it doesn't work. It seem not to be public.

**Reply by ApetersDev** [reply]
sorry, permission was set to private. please try again
